tools/release/java/build.xml

```diff
@@ -62,29 +58,56 @@
             <arg value = "${newdir}"/>
 
             <arg value = "--cppxslt"/>
-            <arg value = "${basedir}/src/com/ibm/icu/dev/tools/docs/dumpAllCppFunc.xslt"/>
+            <arg value = "${rsrc.dir}/com/ibm/icu/dev/tools/docs/dumpAllCppFunc.xslt"/>
 
             <arg value = "--cxslt"/>
-            <arg value = "${basedir}/src/com/ibm/icu/dev/tools/docs/dumpAllCFunc.xslt"/>
+            <arg value = "${rsrc.dir}/com/ibm/icu/dev/tools/docs/dumpAllCFunc.xslt"/>
 
             <arg value = "--reportxslt"/>
-            <arg value = "${basedir}/src/com/ibm/icu/dev/tools/docs/genReport.xslt"/>
+            <arg value = "${rsrc.dir}/com/ibm/icu/dev/tools/docs/genReport.xslt"/>
 
             <arg value = "--resultfile"/>
             <arg value = "${basedir}/APIChangeReport.html"/>
 
-            <classpath>
-                <pathelement location="${build.dir}"/>
-                <pathelement path="${java.class.path}/"/>
-                <pathelement location="${lib.dir}/xalan.jar"/>
-		<fileset dir="${lib.dir}"/>
-                <pathelement location="${lib.dir}/xercesImpl.jar"/>
-                <pathelement path="."/>
-            </classpath>
+        </java>
+    </target>
+
+    <target name="apireport_md" depends="tools">
+        <java jar="${apireport.jar}" fork="yes" failonerror="true">
+
+<!--
+            <arg value = "-DELETEME-newver"/>
```

Comment:
could you use properties here?

Please add some user level comments describing when/why someone would set these.

Reply:
this is preexisting. it overwrites the version number rather than have it calculated (if the TC wants to change the title to a specific version number) I can document though.

Reply:
I'm just dropping the comments.